client/my-sites/site-settings/sitemaps.jsx

```diff
@@ -0,0 +1,236 @@
+
+class Sitemaps extends Component {
+	static defaultProps = {
+		isSavingSettings: false,
+		isRequestingSettings: true,
+		fields: {},
+	};
+
+	static propTypes = {
+		isSavingSettings: PropTypes.bool,
+		isRequestingSettings: PropTypes.bool,
+		fields: PropTypes.object,
+	};
+
+	isSitePublic() {
+		const { fields } = this.props;
+		return parseInt( fields.blog_public ) === 1;
+	}
+
+	renderSitemapLink( sitemapUrl ) {
+		return (
+			<div key={ sitemapUrl }>
+				<ExternalLink href={ sitemapUrl } icon target="_blank">
+					{ sitemapUrl }
+				</ExternalLink>
+			</div>
+		);
+	}
+
+	renderInfoLink( url ) {
+		const { translate } = this.props;
+
+		return (
+			<div className="sitemaps__info-link-container site-settings__info-link-container">
+				<InfoPopover position={ 'left' }>
+					<ExternalLink href={ url } icon target="_blank">
+						{ translate( 'Learn more about Sitemaps.' ) }
+					</ExternalLink>
+				</InfoPopover>
+			</div>
+		);
+	}
+
+	renderSitemapExplanation() {
+		const { translate } = this.props;
+
+		return (
+			<FormSettingExplanation>
+				{ translate( 'Your sitemap is automatically sent to all major search engines for indexing.' ) }
+			</FormSettingExplanation>
+		);
+	}
+
+	renderNonPublicExplanation() {
+		const {
+			siteSlug,
+			translate
+		} = this.props;
+
+		return (
+			<FormSettingExplanation>
+				{
+					translate(
+						'Your site is currently not accessible to search engines. ' +
+						'It is possible that "Privacy" is not set to "Public" in your {{a}}General{{/a}} settings.',
+						{
+							components: {
+								a: <a href={ '/settings/general/' + siteSlug } />,
+							}
+						}
+					)
+				}
+			</FormSettingExplanation>
+		);
+	}
+
+	renderWpcomSettings() {
+		const { site } = this.props;
+		const sitemapTypes = [
+			'sitemap',
+			'news-sitemap',
+		];
+
+		return (
+			<div>
+				{ this.renderInfoLink( 'https://support.wordpress.com/sitemaps/' ) }
+
+				{ this.isSitePublic()
+					? (
+						<div>
+							{ this.renderSitemapExplanation() }
+							{
+								sitemapTypes.map( ( sitemapType ) => {
+									const sitemapUrl = site.URL + '/' + sitemapType + '.xml';
+									return this.renderSitemapLink( sitemapUrl );
+								} )
+							}
+						</div>
+					)
+					: this.renderNonPublicExplanation()
+				}
+			</div>
+		);
+	}
+
+	renderJetpackSettingsContent() {
+		const {
+			activatingSitemapsModule,
+			sitemapsModule,
+			sitemapsModuleActive,
+			translate
+		} = this.props;
+
+		if ( ! activatingSitemapsModule && ! sitemapsModuleActive ) {
+			return;
+		}
+
+		const sitemapTypes = [
+			'sitemap_url',
+			'news_sitemap_url',
+		];
+		const isSitePublic = this.isSitePublic();
+
+		return (
+			<div className="sitemaps__module-settings site-settings__child-settings">
+				{ ! isSitePublic && this.renderNonPublicExplanation() }
+
+				{
+					isSitePublic && activatingSitemapsModule &&
+					<FormSettingExplanation>
+						{ translate( 'Generating sitemap…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Generating...' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings-sitemaps-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).